quantum/process_keycode/process_space_cadet.c

```diff
@@ -91,7 +94,12 @@ void perform_space_cadet(keyrecord_t *record, uint8_t holdMod, uint8_t tapMod, u
     }
   }
   else {
-    if (sc_last == holdMod && timer_elapsed(sc_timer) < TAPPING_TERM) {
+#ifdef TAPPING_TERM_PER_KEY
+    if (sc_last == holdMod && timer_elapsed(sc_timer) < get_tapping_term(keycode))
```

Comment:
Just for clarification, `get_tapping_term` takes a uint16_t while `keycode` here is uint8_t. Should `get_tapping_term` expect the `holdMod` at all?

Reply:
I did test this, and it did compile fine.  But I could re-type the keycode, to ensure that.... But I don't think it's necessary. 

But .... yeah, this is grabbing the wrong keycode.  I'll need to fix this, so it's passing on the uint16_t keycode, too.  And check that instead. 

AKA, it should be checking `KC_LSPO` specifically.

Reply:
Sorry, i didnt mean to infer a compilation issue. Just wasnt sure of the names we give keycodes in u8 vs u16, i want to say basic and extended?

Reply:
No, you didn't.  It's what I get for doing stuff late at night.  

I didn't check the code.  The `keycode` in that function is actually just the keycode that gets pressed for the space cadet stuff, and is NOT what should be checked, since it's not actually the space cadet keycode.